enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -12,9 +12,11 @@
 from ipython_genutils.importstring import import_item
 from notebook.services.kernels.kernelmanager import MappingKernelManager
 from jupyter_client.ioloop.manager import IOLoopKernelManager
+from traitlets import log as traitlets_log
 
 from ..processproxies.processproxy import LocalProcessProxy, RemoteProcessProxy
 from ..sessions.kernelsessionmanager import KernelSessionManager
+from enterprise_gateway.mixins import EnterpriseGatewayConfigMixin
```

Comment:
Do y'all have an opinion on relative vs absolute imports? Mixing is bad, so I'll probably change this. That said, I really don't like the `...mixins` import

Reply:
I retract that actually. Mixing is guaranteed to happen when you're using relative imports mostly given some of them are going to need a lot of dots. It's a cosmetic concern at worst.